basics_scenarios/sitewise_scenario/SPECIFICATION.md

```diff
@@ -0,0 +1,336 @@
+# AWS IoT SiteWise Service Scenario Specification
+
+## Overview
+This SDK Basics scenario demonstrates how to interact with AWS IoT SiteWise using an AWS SDK. 
+It demonstrates various tasks such as creating a SiteWise Asset Model, creating an asset, 
+sending data to the asset, reading data, and so on.  Finally this scenario demonstrates how 
+to clean up resources. Its purpose is to demonstrate how to get up and running with 
+AWS IoT SiteWise and an AWS SDK.
+
+## Resources
+This Basics scenario requires an IAM role that has permissions to work with AWS IoT 
+SiteWise service. The scenario creates this resource using a CloudFormation template.
+
+## Hello AWS IoT SiteWise
+This program is intended for users not familiar with the AWS IoT SiteWise Service to easily get up and running. The logic is to show use of 
+`listAssetsPaginator`.
+
+## Basics Scenario Program Flow
+The AWS IoT SiteWise Basics scenario executes the following operations.
+
+1. **Create an AWS SiteWise Asset Model**:
+   - Description: This step creates an AWS SiteWise Asset Model by invoking the `createAssetModel` method.
+   - Exception Handling: Check to see if a `ResourceAlreadyExistsException` is thrown. 
+     If it is thrown, get the asset model ID and move on.
+
+2. **Create an AWS IoT SiteWise Asset**:
+   - Description: This operation creates an AWS SiteWise asset.
+   - The method `createAsset` is called to obtain the asset ID.
+   - Exception Handling: Check to see if a `ResourceNotFoundException` is thrown. If so, 
+   display the message and end the program.
+
+3. **Retrieve the property ID values**:
+   - Description: To send data to an asset, we need to get the property ID values for the 
+   the model properties. This scenario uses temperature and humidity properties.
+   - The method `listAssetModelProperties()` is called to retrieve the property ID values.
```

Comment:
Either remove all `()` for methods or add them to all. Right now it's inconsistent.

Reply:
I removed them from spec description sections.